### ISSUE TITLE
Weather Symbols With Height Value

### ISSUE THREAD

**Issue by david-melendez** [missioncommand/old-mil-sym-java#46]
There are a few weather symbols that have a height value associated with them.

- Freezing Levels
- Tropopause Level
- Tropopause Low
- Tropopause High

Here is the text from the MIL-STD-2525C standard.

```
METOC.AMPHC.PRS.LOWCTR.TROPLW

METOC
    ATMOSPHERIC
        PRESSURE SYSTEMS
            LOW PRESSURE CENTER
                TROPOPAUSE LOW

Hierarchy: 3.1.1.1.2

Parameters:
1. Anchor Points. This graphic requires one anchor point. The point defines the geometric center of the graphic.

2. Size/Shape. Not applicable.

3. Orientation. The graphic is oriented on the display as shown in the example and is
operator-centered over the desired location. The center of the graphic is the pressure
center. The low point of the tropopause topography is indicated by the letter L and
height above mean sea level is included within the graphic.
```

Notice the last line says that the height above sea level is included within the graphic.  My question is how to get that value sent to the renderer so that value is actually included in the graphic.

**Comment by michael-spinelli** [comment]
Not currently implemented.  I'll look into fixing that.

**Comment by michael-spinelli** [comment]
fixed in release v0.1.19

modifier to use is X_ALTITUDE_DEPTH
